anchore_manager/util/proc.py

```diff
@@ -0,0 +1,45 @@
+"""
+Operations for process things like exit codes, forking etc
+"""
+
+import enum
+import sys
+
+
+class ExitCode(enum.Enum):
+    """
+    Exit codes and conditions for consistency in the CLI
+    """
+
+    ok = 0
+    failed = 2
+    obj_store_failed = 5 # Special value for compat with the object store ops return codes
+
+
+def doexit(ecode: ExitCode):
+    """
+    Robust exit, closing stdout and stderr
+
+    :param ecode:
+    :return:
+    """
+
+    try:
+        sys.stdout.close()
+    except:
+        pass
+    try:
+        sys.stderr.close()
```

Comment:
this messes up testing the CLI with `click` (see https://github.com/pallets/click/issues/824 )

What is the reason we need to have this implemented here? I encountered the problems in the anchore-cli (see pr https://github.com/anchore/anchore-cli/pull/66 )

Reply:
I honestly don't recall. This is moved code from how it was previously implemented and I didn't want to mess with it much. I'm happy to review our CLI approaches in general once this release is out.

Reply:
@nurmi do you recall why this is here?

Reply:
I'm going to leave it for now since this is not a change from current behavior, but happy to change it in the future.